### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by lpsinger**
This looks a little bit nicer than the HEASARC form.

https://user-images.githubusercontent.com/728407/230526591-cdd024a8-8c03-42d0-8170-11e0bf74bf19.mov

### CODE REVIEW

**Review by dakota002 — APPROVED** [review]
Looks great!

File: app/routes/contact.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  })
+
+  let recipient
+  switch (service) {
+    case 'gcn':
+      recipient = 'gcnkafka@lists.nasa.gov'
+      break
+    case 'gcn-classic':
+      recipient = 'gcnclassic@lists.nasa.gov'
+      break
```
Comment (lpsinger) [inline app/routes/contact.tsx:51]: @jracusin, are these email addresses correct?

Comment (jracusin) [reply app/routes/contact.tsx:51]:
```suggestion
    case 'gcn':
      recipient = 'gcnkafka@lists.nasa.gov'
      break
    case 'gcn-classic':
      recipient = 'gcn-classic@athena.gsfc.nasa.gov'
      break
```

Comment (lpsinger) [reply app/routes/contact.tsx:51]: Thanks. Fixed in f87ce9e5ce4593b17fea72b8b948d86e8cd8a1a1.

**Comment by Vidushi-GitHub** [comment]
Looks nice!

I have a few comments:
(a) Need more description about GCN and GCN classic
(b) Instead of What is you email address?, What is your question about?, better words can be used.

**Comment by lpsinger** [comment]
> (b) Instead of What is you email address?, What is your question about?, better words can be used.

Would you open a PR with better wording?

**Pull request merged**